cmd/restic/cmd_backup.go

```diff
@@ -482,6 +482,7 @@ func runBackup(ctx context.Context, opts BackupOptions, gopts GlobalOptions, ter
 	var err error
 
 	if runtime.GOOS == "windows" {
+		fs.EnableBackupPrivilege()
```

Comment:
This doesn't compile on non-windows systems. I'd prefer to let `internal/fs/fs_local.go` handle the privilege request automatically. It should request the privileges when the fs instance is created.